### ISSUE TITLE
Plugin compatibility issues

### ISSUE THREAD

**Issue by pupi1985** [q2apro/q2apro-quizx#1]
I like the initiative and I'm pretty sure many users will share the feeling too. However, there are a few things that I think should have priority 1 to fix.

[This](https://github.com/q2apro/q2apro-quizx/blob/master/quizx-layer.php#L18) requires a call to the parent method. Otherwise, this plugin brakes the chain and subsequent plugins never have this method called. (Or even better, just remove the empty method).

[Here](https://github.com/q2apro/q2apro-quizx/blob/master/quizx-layer.php#L1390) happens the same thing: the full method is overriden and there no parent call. So any other plugin running after this one will have that method not executed.

[Here](https://github.com/q2apro/q2apro-quizx/blob/master/quizx-layer.php#L1422) there is conditional logic that may block the execution of that method. When it goes through the `else` everything goes fine. When in game mode, chain is broken. Every possible case must be taken into account to make sure a parent method is called.

Same [here](https://github.com/q2apro/q2apro-quizx/blob/master/quizx-layer.php#L1505) and in some methods below which I'm not linking.

In short, every method overridden (present in the base theme or even in the theme) must call the parent one. Otherwise, plugin and theme conflicts will arise and the other plugins will seem to be the faulty ones (e.g.: due to lack of initialization).

Doing so might require a lot of refactoring some times. Otherwise it should be a good idea to clarify what methods in the layer will generate conflicts with other plugins. That way it is up to the user to decide what to do and also serves as a good disclaimer.

**Comment by q2apro** [comment]
Thanks for spoting those points. I was developing the Quizx plugin as a standalone, just taking default plugins into consideration and the default SnowFlat theme.

Feel free to add/pull those changes.
